libs/wingsdk/src/target-tf-aws/function.ts

```diff
@@ -127,6 +127,8 @@ export class Function extends cloud.Function implements IAwsFunction {
       role: this.role.name,
       policy: Lazy.stringValue({
         produce: () => {
+          this.policyStatements = this.policyStatements ?? [];
```

Comment:
The rest of this function no longer needs `this.policyStatements?.` or `this.policyStatements ?? []`

Reply:
yes, you are right.